ui/apps/platform/src/Containers/Collections/CollectionResults.tsx

```diff
@@ -214,29 +194,24 @@ function CollectionResults({
                 >
                     {isRefreshingResults ? (
                         <>
-                            {deployments.map((deployment: ListDeployment) => (
-                                <DeploymentSkeleton key={`refreshing-${deployment.id}`} />
+                            {Array.from(Array(10).keys()).map((index: number) => (
```

Comment:
Is it a correct understanding that while refreshing, render 10 deployment skeletons independent of how many deployment result items? That makes sense to me.

Reply:
Yes I meant to comment on this in the PR.

This is primarily driven by the new hook, since the data lifecycle that makes sense in the general case makes it hard to keep a a hold of the old data to use as a base for the skeletons.

Regardless though, I think rendering one page worth (10) of skeletons is probably a better user experience than rendering an arbitrary number of skeletons based on previous data (especially if the previous request returned zero results.)